lyrics/rock/november-rain.txt

```diff
@@ -49,26 +49,10 @@ When there's no one left to blame
 So never mind the darkness
 We still can find a way
 'Cause nothin' lasts forever
-Even cold November rain
+Even cold August rain
 
 Don't ya think that you need somebody
 Don't ya think that you need someone
 Everybody needs somebody
-You're not the only one
-You're not the only one
-
-Don't ya think that you need somebody
-Don't ya think that you need someone
-Everybody needs somebody
-You're not the only one
-You're not the only one
-
-Don't ya think that you need somebody
-Don't ya think that you need someone
-Everybody needs somebody
-You're not the only one
-You're not the only one
-
-Don't ya think that you need somebody
-Don't ya think that you need someone
-Everybody needs somebody
\ No newline at end of file
+Cosa más bella que tú
+Cosa más linda que tú

```

Comment:
Are you sure @kbjarkefur that Gun's and Roses gets better by mixing in Eros Ramazzotti?